features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -137,6 +136,10 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
     const [ legacyPasswordPolicies, setLegacyPasswordPolicies ] = useState<ConnectorPropertyInterface[]>([]);
 
     const isReadOnly: boolean = !useRequiredScopes(featureConfig?.governanceConnectors?.scopes?.update);
+    const hasScopesForRuleBasedPasswordExpiry: boolean =
+        useRequiredScopes(ValidationConfigConstants.RULE_BASED_PASSWORD_EXPIRY_REQUIRED_SCOPES);
+    const isRuleBasedPasswordExpiryDisabled: boolean = disabledFeatures?.includes("ruleBasedPasswordExpiry")
```

Comment:
Let's add `ruleBasedPasswordExpiry` to the feature dictionary and use the constant here.

Ref. impl: https://github.com/wso2/identity-apps/blob/c3dc6bf38203b4e73f3b877382e55eaf7995b3be/features/admin.applications.v1/constants/application-management.ts#L70-L88